tex/chProgramming.tex

```diff
@@ -969,7 +977,7 @@ \subsection{Natural numbers}\index[concept]{natural number}
 Definition same_bool b1 b2 :=
   match b1 with
   | true => match b2 with true => true | _ => false end
-  | false => match b2 with true => false | _ => true end
+  | false => match b2 with false => true | _ => false end
```

Comment:
I am against applying this change: The above sentence says "The above code defining same_bool is parsed as follows".  If you use Print to look at the code, what you obtain is written with if-then-else constructs, but because it displays the value for true first, the original text is more faithful.

Reply:
Aaah, good point. Starting with "true" is more canonical after all.